packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -492,18 +492,18 @@ const SwipeableDrawer = React.forwardRef(function SwipeableDrawer(inProps, ref)
     if (variant === 'temporary') {
       const doc = ownerDocument(paperRef.current);
       doc.addEventListener('touchstart', handleBodyTouchStart);
-      doc.addEventListener('touchmove', handleBodyTouchMove, { passive: false });
+      doc.addEventListener('touchmove', handleBodyTouchMove, { passive: !open });
```

Comment:
```suggestion
      // A blocking listener prevents Firefox's navbar to auto-hide on scroll.
      // It only needs to prevent scrolling on the drawer's content when open.
      // When closed, the overlay prevents scrolling.
      doc.addEventListener('touchmove', handleBodyTouchMove, { passive: !open });
```